src/authorization/WebAclReader.ts

```diff
@@ -50,9 +50,11 @@ export class WebAclAuthorizer extends Authorizer {
     this.accessChecker = accessChecker;
   }
 
-  public async canHandle({ identifier }: AuthorizerArgs): Promise<void> {
-    if (this.aclStrategy.isAuxiliaryIdentifier(identifier)) {
-      throw new NotImplementedHttpError('WebAclAuthorizer does not support permissions on auxiliary resources.');
+  public async canHandle({ credentials }: PermissionReaderInput): Promise<void> {
+    for (const [ key, value ] of Object.entries(credentials)) {
+      if (value && key !== AGENT && key !== EVERYONE) {
```

Comment:
I also talk a bit about this above, but there is no way to determine the groups based on the typing. We could set a stricter type in the method signature here, but since we just chain these objects through components.js and the LDP handler that calls them is not aware of these typings they would have no effect here.

Reply:
Fair enough; I'd still consider `undefined` though for those.